remote-query/remote-query-client/src/main/java/org/infinispan/query/remote/client/impl/QueryRequest.java

```diff
@@ -110,6 +122,7 @@ public QueryRequest readFrom(ProtoStreamReader reader) throws IOException {
          queryRequest.setStartOffset(reader.readLong("startOffset"));
          queryRequest.setMaxResults(reader.readInt("maxResults"));
          queryRequest.setNamedParameters(reader.readCollection("namedParameters", new ArrayList<>(), NamedParameter.class));
+         queryRequest.setLocal(reader.readBoolean("local"));
```

Comment:
why don't use the constant?
```suggestion
         queryRequest.setLocal(reader.readBoolean(LOCAL_FIELD));
```

Reply:
ok, will move the other fields to use constant too